roles/confluent.variables/vars/main.yml

```diff
@@ -312,6 +312,10 @@ kafka_broker_properties:
                             kafka_broker_truststore_path, kafka_broker_truststore_storepass, kafka_broker_keystore_path, kafka_broker_keystore_storepass, kafka_broker_keystore_keypass,
                             plain_jaas_config, kafka_broker_keytab_path, kafka_broker_kerberos_principal|default('kafka'), kerberos_kafka_broker_primary,
                             sasl_scram_users.admin.principal, sasl_scram_users.admin.password, rbac_enabled_public_pem_path ) }}"
+  controlplane_listener:
+    enabled: "{{ kafka_broker_configure_control_plane_listener|bool and kafka_broker_configure_multiple_listeners|bool }}"
```

Comment:
So the comment made me think this property should be enabled always, now without the comment this is fine. but just want to make sure

Reply:
I'll rephrase

Should `control.plane.listener.name` be set in the server.properties no matter what? OR set the property only when you enable it?

Reply:
Take `inter.broker.listener.name` as an example, it is under

```
  defaults:
    enabled: true
    properties:
      inter.broker.listener.name: "{{kafka_broker_listeners[kafka_broker_inter_broker_listener_name]['name']}}"
```

Which means it will always get set. This one is only conditionally `enabled` and I'm wondering if you intended for it be always enabled? IE move it up in the same sub dictionary as `inter.broker.listener.name`

Reply:
No, It is not my intention to have control.plane.listener.name always enabled. From KIP-291 documentation

>  Whenever the "control.plane.listener.name" is set, upon broker startup, we will validate its value and make sure it's different from the inter-broker-listener-name value.
>     If the "control.plane.listener.name" config is not set, the controller will fall back to the current behavior and use inter-broker-listener-name value to determine controller-to-broker endpoints.

Therefore I think it is better to have it conditionally enabled.